proto/buf/alpha/registry/v1alpha1/plugin_curation.proto

```diff
@@ -135,8 +132,7 @@ message CuratedPlugin {
   // the algorithm.
   // Ref: https://github.com/opencontainers/image-spec/blob/main/descriptor.md#digests
   string container_image_digest = 6;
-  // List of default plugin options in the form "{option}={defaultOption}"
-  repeated string default_options = 7;
+  reserved 7;
```

Comment:
I'm okay removing this entirely to avoid confusion. This is an unreleased feature and the only consumer is us.

If you do keep this, then we typically also include the field name, e.g.,

https://github.com/bufbuild/buf/blob/2b2aa72a9f3c8b73c807a650c556d8ec49c4c347/proto/buf/alpha/module/v1alpha1/module.proto#L69-L70

Reply:
I'll just remove it - I didn't want to reshuffle all the field ids and just reserved the id to make it clear it could be used in a follow up if we wanted, but didn't reserve the JSON field name b/c I didn't think we'd rely on JSON serialization anywhere. Probably overthinking it.